src/components/MessageList/MessageNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, MouseEventHandler } from 'react';
+
+export interface MessageNotificationProps {
+  children: React.ReactNode;
+  /** button click event handler */
+  onClick: MouseEventHandler;
+  /** Wether or not to show notification */
+  showNotification: boolean;
+}
+
+const UnMemoizedMessageNotification: FC<MessageNotificationProps> = ({
+  children,
+  onClick,
+  showNotification,
+}) => {
```

Comment:
trying to destructure props on the first line of the body instead of in the argument itself. and we don't need to do the defaultProps on line 31 and can default it here
```suggestion
const UnMemoizedMessageNotification: React.FC<MessageNotificationProps> = (props) => {
  const {
    children,
    onClick,
    showNotification = true,
  } = props;
```